Makefile

```diff
@@ -52,7 +56,7 @@ check-clean-work-tree:
 .PHONY: build
 build:
 	# TODO: Fix this on windows.
-	set -e; for dir in $(ALL_GO_MOD_DIRS); do \
+	set -e; for dir in $(filter-out $(TOOLS_MOD_DIR), $(ALL_GO_MOD_DIRS)); do \
```

Comment:
only 'go mod tidy' is run for TOOLS_MOD_DIR. May be it is better to exclude TOOLS_MOD_DIR from ALL_GO_MOD_DIRS and explicitly run 'go mod tidy' on ALL + TOOLS.

Reply:
This is a good idea. The comment about `ALL_GO_MOD_DIRS` could be updated at the same time.